qualtran/simulation/quimb_sim.py

```diff
@@ -120,6 +124,18 @@ def _assign_outgoing(cxn: Connection) -> Soquet:
         if pos is not None:
             fix[tuple([binst])] = pos[binst]
 
+    inds = get_right_and_left_inds(cbloq.signature)
+    if len(inds) != 2:
+        return tn, fix
+
+    l, r = inds
+    for in_soq in l:
+        if in_soq not in tn.ind_map:
```

Comment:
my defensive programming instincts would probably prefer to use `cbloq.connections` and just look for any connection between `LeftDangle` and `RightDangle`. 

I think this would remove the need for the nested for loop as well

Reply:
done